observability-kit-test/observability-kit-agent-it/src/test/java/com/vaadin/observability/test/AbstractViewIT.java

```diff
@@ -0,0 +1,71 @@
+package com.vaadin.observability.test;
+
+import com.vaadin.testbench.IPAddress;
+import com.vaadin.testbench.parallel.ParallelTest;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+
+import java.io.File;
+
+public abstract class AbstractViewIT extends ParallelTest {
+    public static final int SERVER_PORT = Integer
+            .parseInt(System.getProperty("serverPort", "8080"));
+
+    static String hostName;
+    static boolean isHub;
+
+    @BeforeAll
+    public static void setupClass() {
+        String hubHost = System
+                .getProperty("com.vaadin.testbench.Parameters.hubHostname");
```

Comment:
Can `Parameters.getHubHostname()` be used here instead of the hard-coded string?